lib/project_types/theme/themekit.rb

```diff
@@ -41,7 +41,9 @@ def pull(ctx, store:, password:, themeid:, env:)
       def push(ctx, files: nil, flags: nil, remove: false, env:)
         action = remove ? 'remove' : 'deploy'
         command = build_command(action, env)
-        (command << flags << files).flatten!
+        command << flags if flags
+        command << files if files
+        command.flatten!
```

Comment:
Would this be equivalent to `(command << flags << files).compact.flatten!`?

Reply:
Thank you, nice catch!